test/blackbox-tests/test-cases/pkg/opam-repository-download.t

```diff
@@ -35,9 +35,8 @@ Make a mock repo tarball that will get used by dune to download the package
 Our custom cache folder should be populated with the unpacked tarball
 containing the repository:
 
-  $ find fake-xdg-cache | grep HEAD
-  fake-xdg-cache/dune/git-repo/HEAD
-  fake-xdg-cache/dune/git-repo/FETCH_HEAD
+  $ [ -f fake-xdg-cache/dune/git-repo/HEAD ]
+  > [ -f fake-xdg-cache/dune/git-repo/FETCH_HEAD ]
```

Comment:
Oops, thanks for noticing this.

I don't think it makes sense for to test the `FETCH_HEAD`. It was just in the original test because `HEAD` found it and there wasn't enough reasons speaking for excluding it.

But I'll ask a different question: why not just plop in `sort`? I forgot to add it when I revised the test, but it's a rather decent solution that keeps the shell commands fairly readable and the cram test meaningful.

Reply:
@Leonidas-from-XIV sort also works too. I thought this would be simpler as it directly checks the existence rather than relying on `find`.